### PR TITLE
fix some typos in readme

### PR DESCRIPTION
**Pull request by cAttte**
thank me later!

### CODE REVIEW

**Review by arc25275 — CHANGES_REQUESTED** [review]
Holy fucking shit this is so bad

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-## Hi, I'm arc!
+## Hi, I'm an idiot!
```
Comment (arc25275) [inline README.md]:
```suggestion
## Hi, I'm an stupid fucking idiot!
```
Good idea

**Comment by cAttte** [comment]
damn

**Review by arc25275 — COMMENTED** [review]
A change that would be good

File: README.md (hunk 1 above)
Comment (cAttte) [reply README.md]: ooh yeah

**Comment by arc25275** [comment]
Okay thanks!  I'll merge this now!

**Review by arc25275 — APPROVED** [review]

**Comment by cAttte** [comment]
coward

**Comment by arc25275** [comment]
Ok nerd

**Pull request closed without merging**